internal/cloudfront/distribution_test.go

```diff
@@ -66,7 +68,7 @@ func (s *DistributionTestSuite) TestDistributionBuilder_New() {
 	priceClass := "test price class"
 	group := "test group"
 
-	dist, err := cloudfront.NewDistributionBuilder(defaultOriginDomain, description, priceClass, group, defaultWebACL).Build()
+	dist, err := cloudfront.NewDistributionBuilder(defaultOriginDomain, description, priceClass, group, defaultWebACL, config.Parse()).Build()
```

Comment:
Calling `config.Parse` here will make the test depend on env vars being set, no? Behavior of the test would also change if we change defaults in viper. 

I'm thinking it would be more straightforward to just instantiate a valid config manually for this test, explicitly. Less coupling with details of _how_ we process the config and just take the config itself as important. Wdyt?

This would also apply to other places here where we call `config.Parse`.

Reply:
Maybe setting a valid config in `SetupSuite` and making it a field of the suite could still keep tests readable with the change I'm suggesting above.

Reply:
Thanks for this! Updated all the tests, setting the config as a suite field and setting the values for each test in a `SetupTest` method.